__tests__/app.test.js

```diff
@@ -214,4 +213,28 @@ describe('/api/articles/:article_id/comments', () => {
             expect(body.msg).toBe('not found')
         })
     })
+})
+
+describe('/api/comments/:comment_id', () => {
+    test('DELETE: 204 responds with appropriate status code and no content', () => {
+        return request(app).delete('/api/comments/18')
+        .expect(204)
+        .then(({ body }) => {
```

Comment:
That empty object is not coming from Express, but from Supertest. Don't need to include it here, only the status code check.